db/db_test.go

```diff
@@ -0,0 +1,25 @@
+package db
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBuildDataSourceName(t *testing.T) {
+	assert := assert.New(t)
```

Comment:
Yeah ik lmao, im wondering why that line is needed, i thought you could just do assert.Equal right way